src/tiled/commanddialog.cpp

```diff
@@ -45,6 +45,9 @@ CommandDialog::CommandDialog(QWidget *parent)
 
     setWindowTitle(tr("Edit Commands"));
     Utils::restoreGeometry(this);
+
+    connect(mUi->pushButton, &QPushButton::clicked, 
+            [this]() { CommandManager::instance()->populateMenu(); });
```

Comment:
Since there are no `Okay` and `Cancel` options, I think disabling the Close (X) button in the CommandDialog would be a simple solution for this.